plugins/right-sizer/cmd/main.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"runtime"
+
+	"github.com/fairwindsops/insights-plugins/right-sizer/src/controller"
+	"github.com/fairwindsops/insights-plugins/right-sizer/src/util"
+	"github.com/golang/glog"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+var opts struct {
+	Verbose int  `env:"VERBOSE" short:"v" long:"verbose" description:"Show verbose debug information"`
+	Version bool `long:"version" description:"Print version information"`
+}
+
+// VERSION represents the current version of the release.
+const VERSION = "v1.2.0"
```

Comment:
I think this should be `0.1.0`.

Maybe we should remove this functionality for now - we have some automated tools for modifying `version.txt` and I think we'd forget to update this.